src/components/SignupsSplit.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTreatments, SplitContext } from '@splitsoftware/splitio-react';
+
+const SignupsSplit = ({
+  testChildren,
+  defaultChildren,
+  treatmentWithConfig,
```

Comment:
Along the lines of making this more general, let's have the split test name and event be props as well.
```suggestion
  testChildren,
  defaultChildren,
  testName,
  testEvent,
```